cmd/podman/common.go

```diff
@@ -90,7 +90,10 @@ func validateFlags(c *cli.Context, flags []cli.Flag) error {
 }
 
 // getContext returns a non-nil, empty context
-func getContext() context.Context {
+func getContext(c *cli.Context) context.Context {
+	if c != nil && c.GlobalBool("trace") {
+		return c.Ctx
```

Comment:
let's also check that `c.Ctx` is not nil.

Reply:
I can add it if you think it's necessary, I just don't see any case in which that condition could have a nil `c.Ctx`.

Reply:
Added extra nil check condition.